docs/lesson-plans/week-1.md

```diff
@@ -102,42 +95,541 @@ every week may not end up helping you as much as it would for our project, and y
-also be encouraging you to extend the application by implementing ideas for improvement 
-that you come up with or things that we suggest but won’t end up covering in our 
-examples. Also, if you decide that you want to take your project in a direction 
-different from where we decide to take ours, or even want to work on your own idea, 
-we’ll be here to help you work through issues and provide you with a lot of the concepts 
-you’ll need to build it.
+database. 
+
+The frontend will use packages such as [Material-UI](https://material-ui.com/) for
+UI components, [Jest](https://jestjs.io/) for testing the frontend, 
+[Axios](https://github.com/axios/axios) as an HTTP client, and [React](https://reactjs.org/).
+
+The backend will use libraries such as [Vert.x and Vert.x-Web](https://vertx.io/),
+a PostgreSQL database driver, [Jackson](https://github.com/FasterXML/jackson)
+JSON utilities, [JUnit 5](https://junit.org/junit5/) for a testing framework, and
+[Mockito](https://site.mockito.org/) for a mocking framework.
+
+Throughout the course, we'll be covering a lot of topics like Git and terminal commands,
+HTML, CSS, and JavaScript, servers, HTTP and REST requests, testing, databases, APIs, and
+cloud deployment. 
+
+Don't worry if any of this seems unfamiliar or new to you, we'll be sure to cover all of this
+and answer any questions you may have.
+
+### How are the projects organized?
+
+When you open up the project root folders, you'll see a bunch of different files and folders,
+probably a lot with strange names and filetypes you've never seen before. We'll go over quick
+descriptions of the files right now. You'll learn a lot more about everything below in the 
+coming weeks, so don't worry if you don't know what we're saying.
+
+#### Common Files
+
+.gitignore is a file containing regular expressions to describe files we don't want Git
+to keep track of when we make commits or push to GitHub.
+
+>A regular expression (regex) is a syntax used to define text matching the given pattern. 
+>Check out [regexr](https://regexr.com/) and [regex101](https://regex101.com/) for some 
+>common regexes, descriptions, and a playground to test some out.
+
+LICENSE is a file detailing licensing information. In our case, we're using the GPL-3.0
+License, which means anyone can use or modify our code for whatever they want (with some
+conditions). You'll rarely have to deal with this in practice, but we wanted to add one
+so that you can see what it looks like.
+
+README.md is sort of like the introduction file. It's written in a format called 
+[Markdown](https://en.wikipedia.org/wiki/Markdown) (the .md filetype), which allows for 
+simple and easy formatting. Usually this file has descriptions of the software, instructions
+for setting it up, and any other information the developer(s) would like to provide to the
+end user.
+
+#### Frontend
+
+node_modules/ is a directory which houses all of the JavaScript packages you'll need. You should
```

Comment:
Like above we might want to give a brief explanation of npm to give context. I'm sure we do that later, but it might be helpful to give some brief details so readers understand why we should (usually) never mess with the directory.

Reply:
1-2 sentence MAX

Reply:
Added.